### PR TITLE
create OVA file with ovftool

### PR DESCRIPTION
**Pull request by trombik**
the previous naive attempt was a failure. you need to create OVA file
for VMware, which is the hypervisor of IDCF cloud.

### CODE REVIEW

**Comment by trombik** [comment]
confirmed the built image runs on IDCF cloud

**Comment by trombik** [comment]
now `vagrant up` successfully boots a VM, confirmed by both FreeBSD 10.3 and OpenBSD 6.0 VMs.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 Creates `.ova` file for `cloudstack` from `vagrant` box.
 
+## Requirements
+
+* ansible
+* VirtualBox
+* vagrant
+* bundler
+* [ovftool](https://my.vmware.com/web/vmware/details?downloadGroup=OVFTOOL400&productId=353) (registration required)
```
Comment (mitsururike) [inline README.md]: the link should be `https://www.vmware.com/support/developer/ovf/` if there is no reason to use the fixed version.

Comment (trombik) [reply README.md]: fixed in 5dc8a03

**Review by mitsururike — APPROVED** [review]

**Pull request merged**